blockchain/chain_sync/src/sync.rs

```diff
@@ -55,15 +55,24 @@ pub struct SyncConfig {
     /// Number of tasks spawned for sync workers
     pub worker_tasks: usize,
 }
-
+impl SyncConfig {
+    pub fn new(req_window: i64, worker_tasks: usize) -> Self {
+        Self {
+            req_window,
+            worker_tasks
+        }
+    }
+}
```

Comment:
I don't think a constructor like this is necessary, but doesn't matter much

Reply:
I can remove if you want and just reference the struct directly in `peer_test.rs` I just defaulted to adding a constructor instead. Give a thumbs up if you want that changed.

Reply:
It's just config, it can change later if more are added (probably would be swapped for a builder pattern or something so don't worry about it)